charts/common/templates/_helpers.tpl

```diff
@@ -4,8 +4,8 @@
 
 {{- define "labels" }}
 app: {{ empty .Values.releaseName | ternary .Release.Name .Values.releaseName }}
-shortname: {{ .Values.shortname }}
```

Comment:
maybe someone already uses `shortname` and we should add it back?

Reply:
Yeah, maybe use a deprecation-window. Leave it for a couple of releases..